_docs-2/troubleshooting/tracing.md

```diff
@@ -16,328 +16,44 @@ enabled follows all the requests made on behalf of the user throughout
-We use the format `row columnFamily:columnQualifier columnVisibility    value`
-(omitting timestamp which records the time an entry is written to the trace table).
-
-Span entries take the following form:
-
-    traceId        "span":(parentSpanId:spanId)            []    spanBinaryEncoding
-    63b318de80de96d1 span:4b8f66077df89de1:3778c6739afe4e1 []    %18;%09;...
-
-The parentSpanId is "" for the root span of a trace.
-The spanBinaryEncoding is a compact Apache Thrift encoding of the original Span object.
-This allows clients (and the Accumulo monitor) to recover all the details of the original Span
-at a later time, by scanning the trace table and decoding the value of span entries
-via `TraceFormatter.getRemoteSpan(entry)`.
-
-The trace table has a formatter class by default (org.apache.accumulo.tracer.TraceFormatter)
-that changes how span entries appear from the Accumulo shell.
-Normal scans to the trace table do not use this formatter representation;
-it exists only to make span entries easier to view inside the Accumulo shell.
-
-Index entries take the following form:
-
-    "idx":service:startTime description:sender  []    traceId:elapsedTime
-    idx:tserver:14f3828f58b startScan:localhost []    63b318de80de96d1:1
-
-The service and sender are set by the first call of each Accumulo process
-(and instrumented client processes) to `DistributedTrace.enable(...)`
-(the sender is autodetected if not specified).
-The description is specified in each span.
-Start time and the elapsed time (start - stop, 1 millisecond in the example above)
-are recorded in milliseconds as long values serialized to a string in hex.
-
-Start time entries take the following form:
-
-    "start":startTime "id":traceId        []    spanBinaryEncoding
-    start:14f3828a351 id:63b318de80de96d1 []    %18;%09;...
-
-The following classes may be run while Accumulo is running to provide insight into trace statistics. These require
-accumulo-trace-VERSION.jar to be provided on the Accumulo classpath (`lib/ext` is fine).
-
-    $ accumulo org.apache.accumulo.tracer.TraceTableStats -u username -p password -i instancename
-    $ accumulo org.apache.accumulo.tracer.TraceDump -u username -p password -i instancename -r
+Accumulo client operations will be traced as part of a client application
+operation if the client application is also instrumented using OpenTelemetry
+and invokes the Accumulo client operation in a Span. Client application
+developers can use the OpenTelemetry [documentation] to instrument the
+application. To collect traces in the client, Accumulo needs the same two
+items mentioned previously: `general.opentelemetry.enabled` in the client
+application and the `io.opentelemetry.api.GlobalOpenTelemetry.globalOpenTelemetry`
+member variable must be set.
 
 ### Tracing from the Shell
 You can enable tracing for operations run from the shell by using the
 `trace on` and `trace off` commands.
 
```

Comment:
Is this still true?

Reply:
Yes. See [this](https://github.com/apache/accumulo/blob/main/core/src/main/java/org/apache/accumulo/core/trace/TraceUtil.java#L56)

Reply:
Ok, so that might not be correct for `clients`. I will update that statement.